pyam/utils.py

```diff
@@ -353,7 +353,9 @@ def _escape_regexp(s):
 
 def years_match(data, years):
     """Return rows where data matches year"""
-    years = [years] if isinstance(years, int) else years
+    years = [years] if (
+        isinstance(years, int) or isinstance(years, np.int64)
```

Comment:
This can be `isinstance(years, [int, np.int64])`  though I think `isinstance(years, np.integer)` [should work](https://stackoverflow.com/questions/37726830/how-to-determine-if-a-number-is-any-type-of-int-core-or-numpy-signed-or-not)

Reply:
First suggestion works, second doesn't seem to (may depend on numpy version)